run_bot.sh

```
@@ -206,28 +269,221 @@ clean_recordings() {
+    # Vérifier les fichiers générés
+    local output_dir="./recordings"
+    if [ -d "$output_dir" ] && [ "$(find $output_dir -name "*.mp4" -o -name "*.wav" | wc -l)" -gt 0 ]; then
+        print_success "✅ Recording files were generated"
+        print_info "Generated files:"
+        find "$output_dir" -name "*.mp4" -o -name "*.wav" | head -5
+    else
+        print_info "ℹ️ No recording files (normal for short test)"
+    fi
+    
+    # Compter les erreurs critiques
+    local critical_errors=$(grep -i "error\|Error\|ERROR" "$log_file" | \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)